### PR TITLE
 Refactor installation instructions according GCP latest UI

### PR DESCRIPTION
**Pull request by brunolnetto**
The main caveat of this refactor are the step count increase, which we may work on to reduce.

### CODE REVIEW

**Comment by brunolnetto** [comment]
The preview is available on repository https://github.com/alloyha/pypinfo

File: README.md [hunk 1]
```diff
@@ -313,50 +312,76 @@ This is relatively painless, I swear.
...
 
 ![create](https://user-images.githubusercontent.com/1324225/47172949-6f4ea880-d315-11e8-8587-8b8117efeae9.png "CREATE PROJECT")
 
-4. This takes you to [https://console.developers.google.com/projectcreate](https://console.developers.google.com/projectcreate). Fill out the form and click CREATE. Any name is fine, but I recommend you choose something to do with PyPI like pypinfo. This way you know what the project is designated for:
+5. This takes you to [https://console.developers.google.com/projectcreate](https://console.developers.google.com/projectcreate). Fill out the form and click CREATE. Any name is fine, but I recommend you choose something to do with PyPI like pypinfo. This way you know what the project is designated for:
 
 ![click](https://user-images.githubusercontent.com/1324225/47173020-986f3900-d315-11e8-90ab-4b2ecd85b88e.png) 
 
-5. The next page should show your new project. If not, reload the page and select from the top menu:
+6. A while after creation, at the left-top corner, select the project name of your choice on dropdown component AND at the left-top corner "Navigation Meny", select option on path 'Cloud Overview > Dashboard':
```
Comment (hugovk) [inline README.md:327]:
```suggestion
6. A while after creation, at the left-top corner, select the project name of your choice on dropdown component AND at the left-top corner "Navigation Menu", select option "Cloud Overview > Dashboard":
```

File: README.md [hunk 2]
```diff
@@ -313,50 +312,76 @@ This is relatively painless, I swear.
...
+9. Enable Big Query API by button "Enable" press: 
+
+![big_query_api](https://user-images.githubusercontent.com/13961685/224558381-4af65bf6-348b-4e48-bd14-d667c4a6e1c7.png)
+
+10. After enabling, click CREATE CREDENTIALS:
 
 ![credentials](https://user-images.githubusercontent.com/1324225/47173432-bc7f4a00-d316-11e8-8152-6a0e6cfab70f.png)
 
-8. Choose the "BigQuery API" and "No, I'm not using them":
+**Remark**: You will be requested to go back to Big Query panel. In this case, click on top-left button "Navigation Menu", option "API and services > Enabled APIs and services" and on consequent page, on item "Big Query API":
```
Comment (hugovk) [inline README.md:349]:
```suggestion
**Note**: You will be requested to go back to Big Query panel. In this case, click on top-left button "Navigation Menu", option "API and services > Enabled APIs and services" and on consequent page, on item "Big Query API": 
```

File: README.md [hunk 3]
```diff
@@ -313,50 +312,76 @@ This is relatively painless, I swear.
...
-10. Click continue and the JSON will download to your computer. Note the download location. Move the file wherever you want:
+13. Select role "BigQuery User" (option path "BigQuery > Big Query User"), press button "Done":
 
-![download](https://user-images.githubusercontent.com/1324225/47173614-331c4780-d317-11e8-9ed2-fc76557a2bf6.png)
+![credentials_page_3](https://user-images.githubusercontent.com/13961685/224557170-73532a10-ad64-4e74-9018-8c5f8ad205d7.png)
 
-11. `pip install pypinfo`
-12. `pypinfo --auth path/to/your_credentials.json`, or set an environment variable
-    `GOOGLE_APPLICATION_CREDENTIALS` that points to the file.
+14. On Big Query API panel (See **Remark** on item *10*), click on tab "CREDENTIALS". On section "Service accounts", click on created credentials on items 11, 12 and 13.
```
Comment (hugovk) [inline README.md:365]:
```suggestion
14. On Big Query API panel (See **Note** on item *10*), click on tab "CREDENTIALS". On section "Service accounts", click on created credentials on items 11, 12 and 13.
```

File: README.md [hunk 4]
```diff
@@ -313,50 +312,76 @@ This is relatively painless, I swear.
...
+10. After enabling, click CREATE CREDENTIALS:
 
 ![credentials](https://user-images.githubusercontent.com/1324225/47173432-bc7f4a00-d316-11e8-8152-6a0e6cfab70f.png)
 
-8. Choose the "BigQuery API" and "No, I'm not using them":
+**Remark**: You will be requested to go back to Big Query panel. In this case, click on top-left button "Navigation Menu", option "API and services > Enabled APIs and services" and on consequent page, on item "Big Query API": 
+
+![enabled_credentials](https://user-images.githubusercontent.com/13961685/224572489-402be9b3-a441-45f0-a469-df3a292b2d80.png)
+
+11. On page from "CREATE CREDENTIALS"-button click, choose the "BigQuery API", "Application Data" and "No, I'm not using them":
```
Comment (hugovk) [inline README.md:353]:
```suggestion
11. On the page after clicking the "CREATE CREDENTIALS" button, choose "BigQuery API", "Application Data" and "No, I'm not using them":
```

File: README.md [hunk 5]
```diff
@@ -313,50 +312,76 @@ This is relatively painless, I swear.
...
+
+![create_credential_key](https://user-images.githubusercontent.com/13961685/224573182-5d812f47-c1c5-4aaa-a774-6ae00ce8250d.png)
+
+16. On appearing box, click on option "JSON" and press button "CREATE": This will start the download of credentials on a JSON file with name pattern `{name}-{credentials_hash}.json`:
+
+![create_private_key](https://user-images.githubusercontent.com/13961685/224573235-70f35826-73bb-4dad-bcbf-e6267d105121.png)
+
+### Installation and authentication
+
+17. Run command `pip install pypinfo` on terminal
```
Comment (hugovk) [inline README.md:379]:
```suggestion
17. Run `python -m pip install pypinfo` in the terminal.
```

**Review by hugovk — CHANGES_REQUESTED** [review]
Thanks! Some minor proofreading suggestions.

**Comment by brunolnetto** [comment]
@hugovk  Done! 

Note: Have you seen this movie? https://www.youtube.com/watch?v=FG9rvNhdOX8

**Review by hugovk — APPROVED** [review]
Thanks!

I didn't double check the steps/screenshots actually match the Google Console, but otherwise looks good!

**Comment by hugovk** [comment]
> @hugovk Done!
> 
> Note: Have you seen this movie? [youtube.com/watch?v=FG9rvNhdOX8](https://www.youtube.com/watch?v=FG9rvNhdOX8)

I have! :D

**Comment by brunolnetto** [comment]
Well, well, well. I think we have a merging PR coming soon on my screen.

**Pull request merged**